code/src/main/java/com/google/sps/storage/data-storage/Business.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.storage;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.time.Instant;
+
+/**
+* This class is used to check if a business was contacted in the last week.
+*/
+@Entity
+public class Business {
+
+  @Id private String placeId;
+  private long timeOfLastContactSeconds;
+
+  // Objecify requires one constructor with no parameters
+  private Business() {}
+  
+  public Business(String placeId) {
```

Comment:
I can see how it is ambiguous since the class is called Business and we don't use a "businessId". I added the comment however it might be better to rename the class.